app/serializers/v1/task_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module V1
+  class TaskSerializer < ActiveModel::Serializer
+
+    attributes :id, :name, :description, :state
```

Comment:
Sounds reasonable. Good enough for me since it is only used for the list which implies the project id. That being said it probably wouldn't be horrible to use the serializer for show and create for consistency in representation. But yes, as you have it, makes total sense.